packages/theming/apple-theme/src/appleColors.types.macos.ts

```diff
@@ -206,26 +206,137 @@ interface FluentUIAppleBasePalette {
 
 /** A set of control tokens used on the FluentUI Apple macOS button */
 interface FluentAppleButtonTokens {
-  brandedBackground: ColorValue;
-  brandBackgroundHovered: ColorValue;
-  brandBackgroundPressed: ColorValue;
   brandForegroundDisabled: ColorValue;
   brandForegroundPressed: ColorValue;
-  neutralStrokeDisabled: ColorValue;
-  neutralBackgroundInverted: ColorValue;
-  neutralForegroundInverted: ColorValue;
+}
+
+interface MacOSAliasColorTokens {
```

Comment:
Is this any different from the AliasColorTokens interface in the theme-types package? If not you should consider pulling the alias colors from that instead of duplicating the interface.